### PR TITLE
[ui-components] refactor: moved actionbuttons

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: packages/ui-components/web/statics/styles/action-buttons.scss [hunk 1]
```diff
@@ -2,44 +2,60 @@
...
+
+.sm {
+  @extend.sm_font_size;
+  line-height: 36px;
+  height: 36px;
+}
+
+.md {
+  @extend.sm_font_size;
+  line-height: 40px;
```
Comment (tamrat-bay) [inline packages/ui-components/web/statics/styles/action-buttons.scss:53]: remove line. height

File: packages/ui-components/web/statics/styles/button.scss [hunk 2]
```diff
@@ -37,3 +38,26 @@
...
+.small_btn_font_size {
+  font-size: 16px;
+}
+
+.tiny {
+  height: 32px;
+}
+
+.small {
+  @extend.small_btn_font_size;
```
Comment (tamrat-bay) [inline packages/ui-components/web/statics/styles/button.scss:51]: make default to 16px and 14 as var

File: packages/plugin-commons/web/src/Components/ColorPicker/CustomColorPickerDialog.jsx [hunk 3]
```diff
@@ -34,16 +34,24 @@ class CustomColorPickerDialog extends Component {
...
+      >
+        <ActionButtons
+          size={BUTTON_SIZE.tiny}
+          isMobile={isMobile}
+          onCancel={this.onCancel}
+          onSave={this.onUpdate}
+          theme={theme}
+          cancelText={t('ColorPickerButtonLabel_Cancel')}
+          saveText={t('ColorPickerButtonLabel_Update')}
+          saveBtnDataHook={'colorPickerUpdateButton'}
```
Comment (shaulgo) [inline packages/plugin-commons/web/src/Components/ColorPicker/CustomColorPickerDialog.jsx:52]: no need to pass dataHook. It should have it hard coded for the actionButtons. Change the test accordingly

File: packages/plugin-video/web/src/toolbar/mediaURLInputModal.tsx [hunk 4]
```diff
@@ -95,6 +95,7 @@ export default class MediaURLInputModal extends Component<Props, State> {
         errorMessage={t('SoundCloudUploadModal_Input_InvalidUrl')}
         placeholder={placeholder}
         onCloseRequested={helpers.closeModal}
+        buttonSize={BUTTON_SIZE.medium}
```
Comment (shaulgo) [inline packages/plugin-video/web/src/toolbar/mediaURLInputModal.tsx:98]: Does urlInputModal have more then one button size?

File: packages/ui-components/web/src/Components/ActionButtons.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  size,
+  onCancel,
+  onSave,
+  cancelText,
+  saveText,
+  isMobile = false,
+  saveBtnDataHook,
+  cancelBtnDataHook,
+  theme,
+  selected = true,
```
Comment (shaulgo) [inline packages/ui-components/web/src/Components/ActionButtons.tsx:32]:
```suggestion
  disableSave,
```

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/plugin-video/web/src/toolbar/mediaURLInputModal.tsx (hunk 4 above)
Comment (tamrat-bay) [reply packages/plugin-video/web/src/toolbar/mediaURLInputModal.tsx:98]: Yes

**Pull request merged**